backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/AccessPackageService.cs

```diff
@@ -75,5 +75,11 @@ public async Task<Dictionary<string, List<AccessPackageDelegation>>> GetDelegati
 
             return sortedAccesses;
         }
+
+        /// <inheritdoc />
+        public Task<HttpResponseMessage> RevokeAccessPackage(Guid from, Guid to, string packageId)
+        {
+            return _accessManagementClient.RevokeAccessPackage(to, from, packageId);
```

Comment:
![Screenshot 2024-12-09 at 11 05 59](https://github.com/user-attachments/assets/e31fe738-ff21-4f9c-9764-010a04f062f6)

It looks like the from and to parameters have swapped places here? 

```suggestion
            return _accessManagementClient.RevokeAccessPackage(from, to, packageId);
```

Reply:
Great catch! 🙌
I switched midway through to always having from first to keep it more consistent with the path in the bff endpoint, but I must have forgotten the Service interface.
Thanks for noticing! 😄